docs/README.md

```diff
@@ -2,26 +2,21 @@
 navTitle: Documentation
 ---
 
-# Welcome to FlowFuse
-
-FlowFuse helps Node-RED developers deliver applications in a more reliable,
-collaborative, and secure manner. Node-RED’s intuitive, low-code development
-environment is great for connecting together hardware devices, APIs, and online
-services.
+# FlowFuse Documentaions
 
 FlowFuse adds to Node-RED; collaborative development, management of
 remote deployments, support for DevOps delivery pipelines, and the ability to
 host Node-RED applications on FlowFuse. FlowFuse is the DevOps platform for
 Node-RED application development and delivery.
 
-## Getting started
+## FlowFuse Manuals
 
- - [FlowFuse Concepts](./user/concepts.md) - Understanding the core terminology and concepts that make up the FlowFuse platform.
- - [User documentation](./user/) - Detailed documentation on how to use FlowFuse.
- - [Migrating from Node-RED to FlowFuse](./user/migration/#migrating-a-node-red-application-to-flowforge) - Read our guide on the steps requireed to move your flows from a standalone Node-RED deployment into a production-ready instance of FlowFuse.
- - [API Documentation](./api/) - Developer API documentation
+ - [User Manual](./user/introduction.md) - Documentation on how to use FlowFuse.
+ - [Administrator Manual](./admin/introduction.md) - How to administer the platform
+ - [API Documentation](./api/)
+ - [Node-RED Migration](./migration/introduction.md) - How to migrate your Node-RED standalone to FlowFuse?
 
-## Device Agent (Edge Compute)
+## Device Agent
```

Comment:
Do we need a paragraph what the agent does? And maybe a pictogram?

Reply:
I think yes, here would also be the place to describe: How to install the Device Agent on different Hardware / PLCs / Gateways etc.